src/main/java/org/opentripplanner/routing/algorithm/mapping/AlertToLegMapper.java

```diff
@@ -96,7 +100,7 @@ public static void addAlertPatchesToLeg(Graph graph, Leg leg, boolean isFirstLeg
             addAlertPatchesToLeg(leg, StopCondition.ARRIVING, alerts, requestedLocale, legStartTime, legEndTime);
         }
 
-        if(leg.isTransitLeg()) {
+        {
```

Comment:
Comment on this block what it does and why it is a block or remove it.

Reply:
These blocks have been removed